packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -413,6 +419,14 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
     }
   }, [state.contentSize, state.virtualizer, state.isAnimating, onLoadMore, isLoading]);
 
+  let keysBefore = [];
+  let key = columnState.currentlyResizingColumn;
+  do {
+    keysBefore.push(key);
+    key = tableState.collection.getKeyBefore(key);
+  } while (key != null);
+  let resizerPosition = keysBefore.reduce((acc, key) => acc + columnState.getColumnWidth(key), 0);
```

Comment:
I think you'll have to account for scroll position here, try going to https://reactspectrum.blob.core.windows.net/reactspectrum/64ad9faffb499da09831a4beeb35e78613d787a2/storybook/index.html?path=/story/tableview--allowsresizing-many-columns-and-rows&providerSwitcher-scale=large&providerSwitcher-toastPosition=bottom and scrolling a bit to the right and then trigger resizing via the column header dropdown. You'll see the indicator is offset from the actual resizer by the amount you scrolled. 

A similar class of issue happens when you drag the column resizer of the first column past the width of the visible table, then trigger the resize mode for the same column via the dropdown. The column will scroll the resizer into view but the nubbin indicator will be in the position the resizer was in before scrolling